Dockerfile

```diff
@@ -1,9 +1,17 @@
+FROM node:alpine as build
+WORKDIR /build
+COPY ./ ./
+RUN npm install gulp
+RUN npm run dist
+
 FROM node:alpine
+# TODO: This needed?
```

Comment:
it was needed when I wrote that. Can you check or remove the comment?

Reply:
I couldn't find `curl` used anywhere in your code when I search for it. I suspect it was simply a remnant from debugging API calls.